controllers/templatesync/template_sync.go

```diff
@@ -1441,7 +1441,27 @@ func (r *PolicyReconciler) emitTemplateEvent(
 		UID:        pol.UID,
 	}
 
-	return sender.SendEvent(ctx, nil, ownerref, policyComplianceReason, msg, compliance)
+	var instance client.Object
+
+	instanceUnstructured := &unstructured.Unstructured{}
+
+	err := instanceUnstructured.UnmarshalJSON(pol.Spec.PolicyTemplates[tIndex].ObjectDefinition.Raw)
+	if err == nil {
```

Comment:
if it is err, we should handle it here seems it sends event when `instance` is nil